### PR TITLE
fix(catalog): catalog/model UI fixes

### PR DESCRIPTION
**Pull request by thewbuk**
Because

- Update states in Pipelines and Models runs
- Prepare for Tags in catalog
- Add tab navigation on Catalog
- Bring back hover tooltip in Catalog
- Add running indication in Pipeline and Model playgrounds
- Fix link in tabs 
- Fix API endpoint from similarity to retrieve

### CODE REVIEW

File: packages/toolkit/src/view/catalog/CatalogMainView.tsx
Comment (EiffelFly) [inline packages/toolkit/src/view/catalog/CatalogMainView.tsx:1]: @thewbuk You have missing dependency, please check your lint

File: packages/toolkit/src/view/catalog/components/CreateCatalogCard.tsx [hunk 1]
```diff
@@ -167,7 +217,7 @@ Tokens: #: ${catalog.totalTokens || "N/A"}
         initialValues={{
           name: catalog.name,
           description: catalog.description,
-          tags: catalog.tags || [],
+          // tags: catalog.tags ? catalog.tags : [],
```
Comment (EiffelFly) [inline packages/toolkit/src/view/catalog/components/CreateCatalogCard.tsx:220]: @thewbuk If this won't be used, let remove it

File: packages/toolkit/src/view/catalog/components/CreateCatalogDialog.tsx
Comment (EiffelFly) [inline packages/toolkit/src/view/catalog/components/CreateCatalogDialog.tsx:1]: @thewbuk Let's remove these comments

File: packages/toolkit/src/view/catalog/components/tabs/CatalogMainTab.tsx
Comment (EiffelFly) [inline packages/toolkit/src/view/catalog/components/tabs/CatalogMainTab.tsx:1]: @thewbuk You have missing lint issue

**Review by EiffelFly — COMMENTED** [review]
@thewbuk Some minor changes

File: packages/toolkit/src/view/catalog/components/CreateCatalogCard.tsx (hunk 1 above)
Comment (thewbuk) [reply packages/toolkit/src/view/catalog/components/CreateCatalogCard.tsx:220]: Tags will be used in a near future

Comment (EiffelFly) [reply packages/toolkit/src/view/catalog/components/CreateCatalogCard.tsx:220]: ok

File: packages/toolkit/src/view/catalog/CatalogMainView.tsx
Comment (EiffelFly) [inline packages/toolkit/src/view/catalog/CatalogMainView.tsx:1]: There still have some hint issues

**Review by EiffelFly — COMMENTED** [review]
@thewbuk Let's fix these hint issue

**Pull request merged**